src/main/java/pageobject_model/test/WebDriverCloudGoogleTest.java

```diff
@@ -35,14 +35,19 @@ public void browserSetup() throws IOException {
     public void calculateMonthlyRentForParameters() throws InterruptedException {
         computeEnginePage.inputNumberOfInstances("4");
         computeEnginePage.optionalPopUpClose();
+
         computeEnginePage.selectDropdownValue("OS","'Free: Debian, CentOS, CoreOS, Ubuntu or BYOL (Bring Your Own License)'");
```

Comment:
OS, machineType, machineType can be moved into separate class as Dropdown values

Reply:
will move to enum inside DropDownObject class